bundles/org.openhab.binding.bluetooth/src/main/java/org/openhab/binding/bluetooth/BluetoothDevice.java

```diff
@@ -102,6 +102,12 @@
      */
     protected String name;
 
+    protected String model;
+    protected String serialNumber;
+    protected String hardwareRevision;
+    protected String firmwareRevision;
+    protected String softwareRevision;
```

Comment:
Could you also add these fields to BluetoothDeviceSnapshot? Please? 😄

Reply:
Didn't notice that you have introduced new snapshot device. What's the idea behind it?

Reply:
It is basically there to keep track of data that was available to discovery participants at the time of discovery. So if new data became available the discovery service would know to rerun the discovery process since discovery participants which were previously unable to discover anything may now be able to discover something.

But on second thought you can skip adding the fields to BluetoothDeviceSnapshot since that should be reserved for data coming from advertising packets.